app/screens/channel_add_people/channel_add_people.tsx

```diff
@@ -0,0 +1,324 @@
+
+    useEffect(() => {
+        mounted.current = true;
+        updateNavigationButtons(false);
+        getProfiles();
+        return () => {
+            mounted.current = false;
+        };
+    }, []);
+
+    useDidUpdate(() => {
+        const canStart = selectedCount > 0 && !startingAddPeople;
+        updateNavigationButtons(canStart);
+    }, [selectedCount > 0, startingAddPeople, updateNavigationButtons]);
+
+    const data = useMemo(() => {
+        if (isSearch) {
+            const exactMatches: UserProfile[] = [];
+            const filterByTerm = (p: UserProfile) => {
+                if (selectedCount > 0 && p.id === currentUserId) {
+                    return false;
+                }
+
+                if (p.username === term || p.username.startsWith(term)) {
+                    exactMatches.push(p);
+                    return false;
+                }
+
+                return true;
+            };
+
+            const results = filterProfilesMatchingTerm(searchResults, term).filter(filterByTerm);
+            return [...exactMatches, ...results];
+        }
+        return profiles;
+    }, [term, isSearch && selectedCount, isSearch && searchResults, profiles]);
+
+    if (startingAddPeople) {
+        return (
+            <View style={style.container}>
+                <Loading color={theme.centerChannelColor}/>
+            </View>
+        );
+    }
+
+    return (
+        <SafeAreaView
+            style={style.container}
+            testID='add_members.screen'
+        >
+            <View style={style.searchBar}>
+                <Search
+                    testID='add_members.search_bar'
+                    placeholder={intl.formatMessage({id: 'search_bar.search', defaultMessage: 'Search'})}
+                    cancelButtonTitle={intl.formatMessage({id: 'mobile.post.cancel', defaultMessage: 'Cancel'})}
+                    placeholderTextColor={changeOpacity(theme.centerChannelColor, 0.5)}
+                    onChangeText={onSearch}
+                    onSubmitEditing={search}
+                    onCancel={clearSearch}
+                    autoCapitalize='none'
+                    keyboardAppearance={getKeyboardAppearanceFromTheme(theme)}
+                    value={term}
+                />
+            </View>
+            {/*
+                https://mattermost.atlassian.net/browse/MM-48489
+                V1 does not have the selected users modal.
+                Add this back in after build the scrollable selectable users panel
+            */}
+            {/* {selectedCount > 0 &&
+                 <SelectedUsers
+                     selectedIds={selectedIds}
+                     warnCount={General.MAX_ADD_USERS - 2}
+                     maxCount={General.MAX_ADD_USERS}
+                     onRemove={handleRemoveProfile}
+                     teammateNameDisplay={teammateNameDisplay}
+                 />
+             } */}
```

Comment:
???

Reply:
selected users panel added and now being use in this PR.